WalletWasabi.Documentation/WasabiCompatibility.md

```diff
@@ -12,11 +12,11 @@ This document lists all the officially supported software and devices by Wasabi
 
 # Officially Supported Hardware Wallets
 
-- ColdCard MK1
```

Comment:
> 3\. it can get people scared who are already using MK1 already with Wasabi, it could make them think we aren't supporting it anymore, while in reality we do

If we do support MK1 then why should we remove it?

Reply:
The MK1 is no longer receiving updates since 2020, there have been multiple security vulnerability reports that lets you access the secret key on the device as far as I am ware, and everyone have upgraded who have a sane mind. So, why should we keep the MK1, even if the current CCK supports it still? The Trezor One is no longer supported by us, but it is a device still supported by Trezor so it is a counter intuitive way of providing security for our users. We should instead prompt our users to upgrade to the latest technology released by coinkite, because the MK1 is not considered secure even by my own standards.